spec/models/manageiq/providers/lenovo/physical_infra_manager/refresher_spec.rb

```diff
@@ -17,6 +32,19 @@
     end
   end
 
+  before(:each) do
+    stub_settings_merge(
```

Comment:
we should be testing the old refresh too, if we are keeping it around

also e.g. on Azure we test that old refresh and new refresh result in the same DB records.

Reply:
it's tested, I'll add it